config/software/nokogiri.rb

```diff
@@ -33,11 +33,11 @@
 
 # nokogiri uses pkg-config, and on a mac that will find the system pkg-config
 # which will find the system pkg-configs which will pull in libicucore from the
-# libxml2 pkg-config spec.  override pkg-configs path here to point into our
-# /opt/chef/embedded pkg-configs.  this should probably be done more generally,
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏This is the fuzziest comment ahaha, it's missing verbs or words, I'm so confused ! It's from the fork so it might have been fixed upstream, I can't understand it now tbh

Reply:
It's originaly from the upstream project, the comment has been there since 2014, I'm not sure I should have updated it, and I'm not sure what it means, nor if it's still relevant

Reply:
Ahaha okk

Reply:
I guess it's fine to leave it like that, it's the modification that made me think you understood it aha